2019-05-13 22:06:08.5664|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The request matched multiple endpoints. Matches: 

Fate.WebApi.Controllers.ValuesController.Get (Fate.WebApi)
Fate.Test.Controllers.ValuesController.Get (Fate.Test)

2019-05-13 22:07:33.5862|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The requested service 'Fate.Common.Repository.Mysql.IRepository`1[[Fate.Domain.Model.Entities.setting, Fate.Domain.Model, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]' has not been registered. To avoid this exception, either register a component to provide the service, check for service registration using IsRegistered(), or use the ResolveOptional() method to resolve an optional dependency.

2019-05-13 22:26:39.3953|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The requested service 'Fate.Common.Repository.Mysql.IRepository`1[[Fate.Domain.Model.Entities.setting, Fate.Domain.Model, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]' has not been registered. To avoid this exception, either register a component to provide the service, check for service registration using IsRegistered(), or use the ResolveOptional() method to resolve an optional dependency.

2019-05-13 22:28:23.7364|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The request matched multiple endpoints. Matches: 

Fate.WebApi.Controllers.ValuesController.Get (Fate.WebApi)
Fate.Test.Controllers.ValuesController.Get (Fate.Test)

2019-05-13 22:47:46.6385|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The request matched multiple endpoints. Matches: 

Fate.WebApi.Controllers.ValuesController.Get (Fate.WebApi)
Fate.Test.Controllers.ValuesController.Get (Fate.Test)

2019-05-13 22:48:40.9723|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The requested service 'Fate.Common.Repository.Mysql.RepositoryBase`1[[Fate.Domain.Model.Entities.setting, Fate.Domain.Model, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]' has not been registered. To avoid this exception, either register a component to provide the service, check for service registration using IsRegistered(), or use the ResolveOptional() method to resolve an optional dependency.

2019-05-13 22:52:08.7788|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The request matched multiple endpoints. Matches: 

Fate.WebApi.Controllers.ValuesController.Get (Fate.WebApi)
Fate.Test.Controllers.ValuesController.Get (Fate.Test)

2019-05-13 22:53:32.7169|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The requested service 'Fate.Common.Repository.Mysql.IRepository' has not been registered. To avoid this exception, either register a component to provide the service, check for service registration using IsRegistered(), or use the ResolveOptional() method to resolve an optional dependency.

2019-05-13 23:12:22.0190|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The request matched multiple endpoints. Matches: 

Fate.WebApi.Controllers.ValuesController.Get (Fate.WebApi)
Fate.Test.Controllers.ValuesController.Get (Fate.Test)

2019-05-13 23:12:22.0190|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The request matched multiple endpoints. Matches: 

Fate.WebApi.Controllers.ValuesController.Get (Fate.WebApi)
Fate.Test.Controllers.ValuesController.Get (Fate.Test)

2019-05-13 23:22:29.2476|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The request matched multiple endpoints. Matches: 

Fate.WebApi.Controllers.ValuesController.Get (Fate.WebApi)
Fate.Test.Controllers.ValuesController.Get (Fate.Test)

2019-05-13 23:22:50.6910|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
111111111111111

2019-05-13 23:23:25.2035|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
111111111111111

2019-05-13 23:40:09.8557|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The request matched multiple endpoints. Matches: 

Fate.WebApi.Controllers.ValuesController.Get (Fate.WebApi)
Fate.Test.Controllers.ValuesController.Get (Fate.Test)

2019-05-13 23:41:27.1141|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
An exception was thrown while activating Fate.Common.Repository.Mysql.RepositoryBase.

2019-05-13 23:41:49.8299|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
An exception was thrown while activating Fate.Common.Repository.Mysql.RepositoryBase.

2019-05-13 23:44:03.3989|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The request matched multiple endpoints. Matches: 

Fate.WebApi.Controllers.ValuesController.Get (Fate.WebApi)
Fate.Test.Controllers.ValuesController.Get (Fate.Test)

2019-05-13 23:44:45.0832|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The request matched multiple endpoints. Matches: 

Fate.WebApi.Controllers.ValuesController.Get (Fate.WebApi)
Fate.Test.Controllers.ValuesController.Get (Fate.Test)

2019-05-13 23:45:49.4486|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
An exception was thrown while activating Fate.Application.Services.SettingApp -> Fate.Domain.Services.Setting -> Fate.Common.Repository.Mysql.UnitOfWork.UnitOfWork -> Fate.Common.Repository.Mysql.RepositoryBase.

2019-05-13 23:46:16.2073|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
An exception was thrown while activating Fate.Application.Services.SettingApp -> Fate.Domain.Services.Setting -> Fate.Common.Repository.Mysql.UnitOfWork.UnitOfWork -> Fate.Common.Repository.Mysql.RepositoryBase.

2019-05-13 23:46:16.9535|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
An exception was thrown while activating Fate.Application.Services.SettingApp -> Fate.Domain.Services.Setting -> Fate.Common.Repository.Mysql.UnitOfWork.UnitOfWork -> Fate.Common.Repository.Mysql.RepositoryBase.

2019-05-13 23:46:17.3236|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
An exception was thrown while activating Fate.Application.Services.SettingApp -> Fate.Domain.Services.Setting -> Fate.Common.Repository.Mysql.UnitOfWork.UnitOfWork -> Fate.Common.Repository.Mysql.RepositoryBase.

2019-05-13 23:46:17.6057|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
An exception was thrown while activating Fate.Application.Services.SettingApp -> Fate.Domain.Services.Setting -> Fate.Common.Repository.Mysql.UnitOfWork.UnitOfWork -> Fate.Common.Repository.Mysql.RepositoryBase.

2019-05-13 23:46:17.8648|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
An exception was thrown while activating Fate.Application.Services.SettingApp -> Fate.Domain.Services.Setting -> Fate.Common.Repository.Mysql.UnitOfWork.UnitOfWork -> Fate.Common.Repository.Mysql.RepositoryBase.

2019-05-13 23:56:26.7615|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
The request matched multiple endpoints. Matches: 

Fate.WebApi.Controllers.ValuesController.Get (Fate.WebApi)
Fate.Test.Controllers.ValuesController.Get (Fate.Test)

2019-05-13 23:56:33.4069|位置：Fate.Common.Middleware.ExceptionHandlerMiddleware+<Invoke>d__3.MoveNext(D:\MyProject\Fate.DDD\Fate.Common\Middleware\ExceptionHandlerMiddleware.cs:43)
111111111111111

